### PR TITLE
Release 3.0.0 edge documentation

### PR DESCRIPTION
**Pull request by resmaa**
**Release 3.0.0
Edge V2 Documentation**

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/919#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @resmaa. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/919#pullrequestreview-1148193522" title="Approved">jgautam</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/919#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04-clusters/03-edge/02-native/01-native-day2.md [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+         passwd: kairos
+  boot:
+    - commands:
+       - modprobe rbd
+
+```
+# Troubleshooting
+
+To be added Troubleshooting  
+
```
Comment (truegilb) [inline content/docs/04-clusters/03-edge/02-native/01-native-day2.md:276]: Either "Coming soon"

File: content/docs/04-clusters/03-edge/02-native.md [hunk 2]
```diff
@@ -15,149 +15,495 @@ import PointsOfInterest from 'shared/components/common/PointOfInterest';
...
+### Staging: Amazon Web Services (AMI)
+
+</Tabs.TabPane>
+
+<Tabs.TabPane tab="KVM" key="KVM">
+
+### Staging: KVM  (QCOW2)
+
+
+ To be added
```
Comment (truegilb) [inline content/docs/04-clusters/03-edge/02-native.md:329]: What is "To be added" under QCOW2?

**Review by truegilb — CHANGES_REQUESTED** [review]

File: content/docs/04-clusters/03-edge/02-native/01-native-day2.md [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+1. Checkout to **this** Git Repository and Switch to  the v2 branch
+Copy the `.local.env.v2.template` file to `.local.env.v2`. Adjust the file settings as follows.
+
+```
+# Not supported yet.
+#PUSH_BUILD="false"
+# Flavor of OS+K8S
+FLAVOR="ubuntu-rke2"
+# Private registry to which the custom image is tagged and can be later pushed.
+IMAGE_REPO="gcr.io/spectro-dev-public/venkat/p6os-ubuntu-rke2"
```
Comment (truegilb) [inline content/docs/04-clusters/03-edge/02-native/01-native-day2.md:78]: We probably should replace Venkat's user id with something like ${YOUR_PATH}

File: content/docs/04-clusters/03-edge/02-native.md (hunk 2 above)
Comment (resmaa) [reply content/docs/04-clusters/03-edge/02-native.md:329]: @jgautam will be updating in the latest.

File: content/docs/04-clusters/03-edge/02-native/01-native-day2.md (hunk 3 above)
Comment (resmaa) [reply content/docs/04-clusters/03-edge/02-native/01-native-day2.md:78]: the entire section 
"Customizing Palette Edge Distribution"
marking this as "to be discussed" as the snippets included are having a lot of Spectro images and other creds. Will work with Gautam on this section.

@jgautam @truegilb

**Review by truegilb — APPROVED** [review]
Approved for now so as to proceed to staging.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/919#pullrequestreview-1148193522" title="Approved">jgautam</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/919#" title="Author self-approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/919#pullrequestreview-1149602521" title="Approved">truegilb</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**